lib/trino-parquet/src/main/java/io/trino/parquet/writer/valuewriter/FixedLenByteArrayShortDecimalValueWriter.java

```diff
@@ -33,7 +33,7 @@ public FixedLenByteArrayShortDecimalValueWriter(ValuesWriter valuesWriter, Type
         super(parquetType, valuesWriter);
         this.decimalType = (DecimalType) requireNonNull(type, "type is null");
         checkArgument(this.decimalType.isShort(), "type is not a short decimal");
-        checkArgument(parquetType.getTypeLength() <= Long.BYTES, "Type length %s exceeded 8 bytes", parquetType.getTypeLength());
+        checkArgument(parquetType.getTypeLength() > 0 && parquetType.getTypeLength() <= Long.BYTES, "Type length %s must be in range 1-8", parquetType.getTypeLength());
```

Comment:
nit: you use `Long.BYTES` for the check, but hardcode `8` in the message. looks slightly weird